Makefile

```diff
@@ -19,7 +19,8 @@ all: build
 build:
 	# Permissions workaround for global `npm install`
 	chown -R root:root /root/st2chatops/
-	npm install --production
+	npm explore npm -g -- npm install node-gyp@latest
+	npm install --production --python=/usr/bin/python3
```

Comment:
It's unclear what's going on and why in these 2 lines. Was this change optional or EL8 can't build without it?
I'm wondering how adding `python3` change to common Makefile affects platforms like EL6/Ubuntu14 which are py2 only?

For example, looking closer at EL6 build revealed this:
```
> iconv@2.2.3 install /root/st2chatops/node_modules/iconv
> node-gyp rebuild

Traceback (most recent call last):
  File "/usr/lib/node_modules/npm/node_modules/npm-lifecycle/node_modules/node-gyp/gyp/gyp_main.py", line 47, in <module>
    import gyp
  File "/usr/lib/node_modules/npm/node_modules/npm-lifecycle/node_modules/node-gyp/gyp/pylib/gyp/__init__.py", line 11, in <module>
    import argparse
ImportError: No module named argparse
gyp ERR! configure error 
gyp ERR! stack Error: `gyp` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onCpExit (/usr/lib/node_modules/npm/node_modules/npm-lifecycle/node_modules/node-gyp/lib/configure.js:351:16)
gyp ERR! stack     at ChildProcess.emit (events.js:198:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:248:12)
gyp ERR! System Linux 4.4.0-96-generic
gyp ERR! command "/usr/bin/node" "/usr/lib/node_modules/npm/node_modules/npm-lifecycle/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /root/st2chatops/node_modules/iconv
gyp ERR! node -v v10.18.1
gyp ERR! node-gyp -v v5.0.7
gyp ERR! not ok 

> node-icu-charset-detector@0.2.0 install /root/st2chatops/node_modules/node-icu-charset-detector
> node-gyp rebuild

Traceback (most recent call last):
  File "/usr/lib/node_modules/npm/node_modules/npm-lifecycle/node_modules/node-gyp/gyp/gyp_main.py", line 47, in <module>
    import gyp
  File "/usr/lib/node_modules/npm/node_modules/npm-lifecycle/node_modules/node-gyp/gyp/pylib/gyp/__init__.py", line 11, in <module>
    import argparse
ImportError: No module named argparse
gyp ERR! configure error 
gyp ERR! stack Error: `gyp` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onCpExit (/usr/lib/node_modules/npm/node_modules/npm-lifecycle/node_modules/node-gyp/lib/configure.js:351:16)
gyp ERR! stack     at ChildProcess.emit (events.js:198:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:248:12)
gyp ERR! System Linux 4.4.0-96-generic
gyp ERR! command "/usr/bin/node" "/usr/lib/node_modules/npm/node_modules/npm-lifecycle/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /root/st2chatops/node_modules/node-icu-charset-detector
gyp ERR! node -v v10.18.1
gyp ERR! node-gyp -v v5.0.7
gyp ERR! not ok 
[            ......] | postinstall:hubot-xmpp: sill doSerial postinstall 4960
[            ......] | postinstall:isstream: info lifecycle isstream@0.1.2~post
[            ......] | postinstall:@slack/client: info lifecycle @slack/client@
[            ......] \ postinstall: info lifecycle hubot-xmpp@0.2.5~postinstall
[             .....] \ postinstall:st2chatops: info lifecycle st2-hubot@0.4.4~p
[             .....] \ postinstall:st2chatops: info lifecycle st2-hubot@0.4.4~p

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: iconv@2.2.3 (node_modules/iconv):
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: iconv@2.2.3 install: `node-gyp rebuild`
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: Exit status 1
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: node-icu-charset-detector@0.2.0 (node_modules/node-icu-charset-detector):
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: node-icu-charset-detector@0.2.0 install: `node-gyp rebuild`
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: Exit status 1
```
Ubuntu14:
```
> iconv@2.2.3 install /root/st2chatops/node_modules/iconv
> node-gyp rebuild

gyp ERR! configure error 
gyp ERR! stack Error: Command failed: /usr/bin/python3 -c import sys; print "%s.%s.%s" % sys.version_info[:3];
gyp ERR! stack   File "<string>", line 1
gyp ERR! stack     import sys; print "%s.%s.%s" % sys.version_info[:3];
gyp ERR! stack                                ^
gyp ERR! stack SyntaxError: invalid syntax
gyp ERR! stack 
gyp ERR! stack     at ChildProcess.exithandler (child_process.js:289:12)
gyp ERR! stack     at ChildProcess.emit (events.js:182:13)
gyp ERR! stack     at maybeClose (internal/child_process.js:962:16)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:251:5)
gyp ERR! System Linux 4.4.0-96-generic
gyp ERR! command "/usr/bin/node" "/usr/lib/node_modules/npm/node_modules/npm-lifecycle/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /root/st2chatops/node_modules/iconv
gyp ERR! node -v v10.14.0
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok 

> node-icu-charset-detector@0.2.0 install /root/st2chatops/node_modules/node-icu-charset-detector
> node-gyp rebuild

gyp ERR! configure error 
gyp ERR! stack Error: Command failed: /usr/bin/python3 -c import sys; print "%s.%s.%s" % sys.version_info[:3];
gyp ERR! stack   File "<string>", line 1
gyp ERR! stack     import sys; print "%s.%s.%s" % sys.version_info[:3];
gyp ERR! stack                                ^
gyp ERR! stack SyntaxError: invalid syntax
gyp ERR! stack 
gyp ERR! stack     at ChildProcess.exithandler (child_process.js:289:12)
gyp ERR! stack     at ChildProcess.emit (events.js:182:13)
gyp ERR! stack     at maybeClose (internal/child_process.js:962:16)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:251:5)
gyp ERR! System Linux 4.4.0-96-generic
gyp ERR! command "/usr/bin/node" "/usr/lib/node_modules/npm/node_modules/npm-lifecycle/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /root/st2chatops/node_modules/node-icu-charset-detector
gyp ERR! node -v v10.14.0
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok
```

Reply:
Good catch. I had that in for testing. I was trying to get the node-gyp compiling to use python3. I have backed those two lines out.